frontend/src/app/AppNotificationDrawer.tsx

```diff
@@ -69,22 +66,18 @@ const AppNotificationDrawer: React.FC<AppNotificationDrawerProps> = ({ onClose }
                   variant={notification.status}
                   title={notification.title}
                 >
-                  <div>
-                    <Button
-                      className="odh-dashboard__notification-drawer__item-remove"
-                      variant={ButtonVariant.plain}
-                      aria-label="remove notification"
-                      onClick={() => onRemoveNotification(notification)}
-                    >
-                      <TimesIcon aria-hidden="true" />
-                    </Button>
-                  </div>
+                  <Button
+                    variant={ButtonVariant.plain}
+                    aria-label="remove notification"
+                    onClick={() => onRemoveNotification(notification)}
+                  >
+                    <TimesIcon aria-hidden="true" />
+                  </Button>
                 </NotificationDrawerListItemHeader>
                 <NotificationDrawerListItemBody
                   timestamp={calculateRelativeTime(notification.timestamp, currentTime)}
-                  className={notification.message ? '' : 'm-is-hidden'}
                 >
-                  {notification.message}
+                  {notification.message ?? undefined}
```

Comment:
What are you doing here?

Reply:
oh sorry, I was trying to make empty string into undefined, I thought it would reduce the body size when there is no message, but I was wrong after testing and forgot to change it back. Will change it back now